apps/shared/account/account.service.ts

```diff
@@ -27,9 +27,19 @@ export class AccountService {
   avatarUrl = DEFAULT_AVATAR_URL;
   /** The display name for the user is available. */
   displayName: string | undefined;
+  /** The current accounts github token, if available. */
+  private githubToken: string | null = null;
 
   constructor(private auth: Auth) {
-    this.auth.onAuthStateChanged((user) => {
+    this.auth.onIdTokenChanged((user) => {
+      if (user === null) {
+        this.githubToken = null;
+      } else {
+        user.getIdTokenResult(false).then((value) => {
```

Comment:
The force refresh parameter is optional right? Maybe omit it, or add a parameter comment for clarity?